cpp/memilio/mobility/metapopulation_mobility_stochastic.h

```diff
@@ -235,7 +237,7 @@ GraphSimulationStochastic<Graph<SimulationNode<Sim>, MigrationEdgeStochastic>>
 make_migration_sim(double t0, double dt, Graph<SimulationNode<Sim>, MigrationEdgeStochastic>&& graph)
 {
     return make_graph_sim_stochastic(t0, dt, std::move(graph), &evolve_model<Sim>,
-                                     &apply_migration<Sim, MigrationEdgeStochastic>);
+                                     &apply_migration<Sim, MigrationEdgeStochastic, true>);
```

Comment:
```suggestion
        static_cast<void (*)(MigrationEdgeStochastic&, size_t, SimulationNode<Sim>&, SimulationNode<Sim>&)>(
            &apply_migration<Sim, MigrationEdgeStochastic>));
```
The issue here is that two apply_migration functions exist in the mio namespace with the same number of template arguments. The number of "normal" arguments cannot be used, since we only pass a function pointer. We don't have to use a cast here, we could also specify the argument EdgeF of make_graph_sim_stochastic.

Reply:
Referring to the discussion above, this looks like a good solution to me. Then we can drop the enable_if.